src/hooks/useStreamingPlanGeneration.ts

```
@@ -219,6 +219,26 @@ export function useStreamingPlanGeneration() {
+                ...prev,
+                status: 'error',
+                planId: prev.planId ?? latestPlanId,
+                error: {
+                  message: event.data.message,
+                  classification: event.data.classification,
+                  retryable: event.data.retryable,
+                },
+              }));
+              const cancelledErr = new Error(
+                event.data.message || 'Plan generation was cancelled.'
+              ) as StreamingError;
```

Biome (2.3.14)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)